app/checklist/views.py

```diff
@@ -88,3 +88,16 @@ def delete_default_checklist_item(id):
         flash('Error occurred. Please try again.', 'form-error')
         return redirect(url_for('checklist.index'))
     return redirect(url_for('checklist.index'))
+
+
+@checklist.route('/<int:id>/view', methods=['GET'])
+def view_all_checklist_items(id):
```

Comment:
need to change this `id` to `user_id` as well

Reply:
Got it!